src/vsr.zig

```diff
@@ -1399,7 +1399,8 @@ pub const Checkpoint = struct {
     }
 
     pub fn valid(op: u64) bool {
-        return op == 0 or (op + 1) % constants.lsm_batch_multiple == 0;
+        return op == 0 or
+            (op + 1) % (constants.journal_slot_count - constants.lsm_batch_multiple) == 0;
```

Comment:
Is there a good name for `constants.journal_slot_count - constants.lsm_batch_multiple`? I'd love to extract this into something like `constants.vsr_checkpoint_op_count`, as this constant captures an important conceptual thing, but I am not 100% positive on the name...

Reply:
I don't think we have a term for it right now. 

Brainstorming:
- "checkpoint span"
- "checkpoint period"

But I think both of those could be confused with the distance between op_checkpoint and the following checkpoint trigger, (even though that is equal to `journal_slot_count`. :thinking: